app/src/main/java/com/example/ithaca_transit_android_v2/Models/MapState.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.ithaca_transit_android_v2.Models
+
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Location
+import java.util.*
+
+sealed class MapState(
+    val trip:Trip = Trip(
+        emptyList(),
+        Location("", Coordinate(0.0, 0.0), ""),
+        Location("", Coordinate(0.0, 0.0), ""),
+        false,
+        Date(0),
+        Date(0),
+        0)
+) {}
```

Comment:
i'm fairly certain that this is just be `sealed class MapState(val trip = Trip ) `

think of `val` as a constant, if you give it a default object then technically the value cannot be modified!